docs/fuel-101/intro-to-utxo.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Intro to UTXO
```

Comment:
Intro to the UTXO Model or just UTXO? I think perhaps the former?

Reply:
I think it should be either "Intro to the UTXO Model" or "Intro to UTXOs"

Reply:
Agreed. What do you prefer?

Reply:
Ahaha I have a bad habit of misaligned titles. I liked "Intro to the UTXO model" the best. Aligned nav, title, and metadata